ConsoleApp/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿byte b1 = 10;
+short s1 = 20;
+int i1 = 30;
+long l1 = 40;
+bool b2 = false;
+float f1 = 50.1f;
+double d1 = 50.2;
+decimal d2 = 50.3m;
+char c1 = '/';
+
+System.Console.WriteLine($"byte:{b1}, short:{s1}, int:{i1}, long:{l1}, bool:{b2}, float:{f1}, double:{d1}, decimal:{d2}, char{c1}");
+
+
+var x = 5;
+var y = 10;
+//-6*x^3+5*x^2-10*x+15
+System.Console.WriteLine(-6 * x ^ 3 + 5 * x ^ 2 - 10 * x + 15);
```

Comment:
x^3 is not the same as Math.Pow(x, 3), it is binary operation (XOR)
use better Math.Pow(x, 3) or x * x * x